main.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,900');
+@import url('https://fonts.googleapis.com/css?family=Raleway:400,600');
+
+html {
+  font-size: 16px;
+}
+body {
+  font-size: 1rem;
+  font-family: 'Roboto', sans-serif;
+}
+
+
+
+/* Programme Builder CELLS */
+
+.prog-cell {
+  border: 1px solid #979797;
```

Comment:
Also needs
`vertical-align: top;` to ensure that sessions that start at the same time are visually inline with one another. At the moment the natural behaviour of a table is to float each cell in the middle of the row which makes them look off kilter.